Sample Data/AIVectraDetect.csv

```diff
@@ -0,0 +1,6 @@
+﻿TimeGenerated,DeviceVendor,DeviceEventClassID,LogSeverity,DestinationIP,DeviceName,SourceIP,DeviceVersion,Activity,DestinationHostName,ExternalID,SourceHostName,SourceUserName,DeviceCustomString4,FlexNumber1,FlexNumber2,AdditionalExtensions,Type
```

Comment:
Making the sample data richer enables the Azure Sentinel community to use this sample data and build richer experiences for your integration in Azure Sentinel which adds more breadth to the current integration. Hence would be great to take care of the following.
1. Does the DeviceEventClassId represent all values of this? Aren't there any other values / types for DeviceEventClassId? If yes, please include those types in sample data as well.
2. Are there other types of activity values you can include in the sample data?
3. Can the AdditionalExtensions field values be mapped to other properties? I see that there are key-values included in this and would get much better value for data by having these values show up under specific fields.
4. Are the device logs that you are surfacing in Azure Sentinel at parity in terms of what's being surfaced in AIVectra product? If no, what are the differences and is it possible to include that data as part of this integration as well?

Reply:
1&2/ I added more sample data in the CSV file. 
3/ The AdditionalExtensions field cannot be mapped to other properties. 
4/ yes, they are in parity

Reply:
Thanks, @danymello - looks great.
Quick check, the data is sanitized, right? No sensitive info, and IPs are fictitious, correct?